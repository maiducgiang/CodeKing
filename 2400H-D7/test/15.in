10
99177 5000
66598 5000
63590 5000
98573 5000
26397 5000
10036 5000
47960 5000
86141 5000
43761 5000
11540 5000
