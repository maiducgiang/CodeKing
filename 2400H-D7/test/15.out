79641836
213698216
840803089
362087320
937648998
80401496
284938770
798276969
776271175
531436620
